pycalphad/core/eqsolver.pyx

```diff
@@ -5,8 +5,6 @@ cimport numpy as np
 cimport cython
 cdef extern from "_isnan.h":
     bint isnan (double) nogil
-import scipy.spatial
```

Comment:
With the removal of this import, pycalphad only depends on `scipy` for the Cython wrappers for LAPACK in `scipy.linalg.cython_lapack`. Specifically, we depend on `cython_lapack.dgelsd()` (least squares) and `cython_lapack.dgesv()` (solve Ax=b).

Reply:
Agreed. In a world of fast network connections and binary packages, the weight of SciPy as a dependency is not a big deal. Plus, users are very likely to already have it installed. In addition, linking LAPACK is apparently a non-trivial and highly platform-dependent activity which I see extremely limited benefit for pycalphad to take on, given that SciPy is already meeting the need.